melodies_monet/driver.py

```diff
@@ -537,6 +555,9 @@ def open_obs(self):
                 o.file = self.control_dict['obs'][obs]['filename']
                 if 'variables' in self.control_dict['obs'][obs].keys():
                     o.variable_dict = self.control_dict['obs'][obs]['variables']
+                if 'drop_unused_vars' in self.control_dict['obs'][obs].keys():
+                    o.drop_unused_vars \
+                        = self.control_dict['obs'][obs]['drop_unused_vars']
```

Comment:
```suggestion
                    o.drop_unused_vars = self.control_dict['obs'][obs]['drop_unused_vars']
```
Many lines in MM are longer than this, don't think need to break